test/engines/wintermute/path_utils.h

```diff
@@ -0,0 +1,229 @@
+#include <cxxtest/TestSuite.h>
+#include "engines/wintermute/utils/path_util.h"
+/**
+ * Test suite for the functions in engines/wintermute/utils/path_util.h
+ *
+ * NOTE: This is not a prescription;
+ *       this was not written by the original engine author;
+ *       this was not written by the engine porter.
+ *
+ * It might, however, help to spot variations in behavior that are introduced by modifications
+ */
+
+class PathUtilTestSuite : public CxxTest::TestSuite {
+	public:
+	const Common::String unixPath;
+	const Common::String unixCapPath;
+	const Common::String windowsPath;
+	const Common::String windowsCapPath;
+	const Common::String emptyString;
+	const Common::String dualExtPath;
+	const Common::String manyExtPath;
+	const Common::String mixedSlashesPath1;
+	const Common::String mixedSlashesPath2;
+	const Common::String unixRelativePath;
+	const Common::String windowsRelativePath;
+	PathUtilTestSuite () :
+		unixPath("/some/file.ext"),
+		unixCapPath("/SOME/FILE.EXT"),
+		windowsPath("C:\\some\\file.ext"),
+		windowsCapPath("C:\\SOME\\FILE.EXT"),
+		emptyString(""),
+		dualExtPath("/some/file.tar.gz"),
+		manyExtPath("/some/file.tar.bz2.gz.zip"),
+		mixedSlashesPath1("C:\\this/IS_REALLY\\weird.exe"),
```

Comment:
Because while absolute paths are the spawn of the devil there are a few games that use them already, by mistake - the CR series seems to be a major offender - and I thought that we might like to test for a defined behavior in that case, just to be sure of what the function is doing in that case.

We might very well want to change that behaviour, of course, but I don't think it has P=0 of happening.